msdata-sqlite/custom-sqlite.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Custom SQLite Versions - Microsoft.Data.Sqlite
+author: bricelam
+ms.date: 11/25/2019
+---
+# Custom SQLite Versions
+
+Microsoft.Data.Sqlite is built on top of SQLitePCL.raw. You can use custom versions of the native SQLite library be using a bundle or by configuring a SQLitePCL.raw provider.
+
+## Bundles
+
+SQLitePCL.raw provides bundle packages that make it easy to bring in the right dependencies across different platforms.
+
+The main Microsoft.Data.Sqlite package brings in SQLitePCLRaw.bundle_e_sqlite3 by default.
+
+To use a different bundle, install the `Microsoft.Data.Sqlite.Core` package along with the bundle package you want to use. Bundles are automatically initialized by Microsoft.Data.Sqlite.
```

Comment:
May want to clarify here that Microsoft.Data.Sqlite.Core is to be installed INSTEAD of Microsoft.Data.Sqlite.  May also want to clarify here that the name .Core has nothing to do with .NET Core?